massa-bootstrap/src/client.rs

```diff
@@ -100,39 +102,52 @@ async fn stream_final_state(
                         }
                     }
                     write_final_state.slot = slot;
+
+                    // Set consensus
+                    if let Some(graph) = global_bootstrap_state.graph.as_mut() {
+                        graph.final_blocks.extend(consensus_part.final_blocks);
+                    } else {
+                        global_bootstrap_state.graph = Some(consensus_part);
+                    }
+                    let last_consensus_step = StreamingStep::Ongoing(
+                        // Note that this unwrap call is safe because of the above conditional statement
+                        global_bootstrap_state
+                            .graph
+                            .as_ref()
+                            .unwrap()
+                            .final_blocks
+                            .iter()
+                            .map(|b_export| b_export.block.id)
+                            .collect(),
+                    );
+
                     // Set new message in case of disconnection
-                    *next_bootstrap_message = BootstrapClientMessage::AskFinalStatePart {
+                    *next_bootstrap_message = BootstrapClientMessage::AskBootstrapPart {
                         last_slot: Some(slot),
                         last_ledger_step,
                         last_pool_step,
                         last_cycle_step,
                         last_credits_step,
                         last_ops_step,
+                        last_consensus_step,
                     };
-                    debug!(
```

Comment:
I removed them because they do not bring any useful information in case of a bootstrap behaviour problem, the logs we need are those are in [`server.rs`](https://github.com/massalabs/massa/blob/consensus-blocks-streaming/massa-bootstrap/src/server.rs#L418)